### ISSUE TITLE
2.2 RC3 import, but no data

### ISSUE THREAD

**Issue by rgazaryants** [OpenTSDB/opentsdb#623]
Importing from 2.1 to 2.2 with tsd.storage.salt.width = 1 and appends enabled from an --import file.
Get stats like it's writing. HBase requests counter matches up to the stats import outputs.
Running a scan on the just imported data yields no results.
Table has data already in it from previous puts that works as expected, but nothing from import ever shows up.

**Comment by rgazaryants** [comment]
It works without salting BTW.

**Comment by rgazaryants** [comment]
For what it's worth, here is the raw data in the table after an import into an empty table with salt width of 1.
```
hbase(main):037:0> scan 'tsdb', LIMIT=>5
ROW                                       COLUMN+CELL
 \x00\xD4V7\xDC\xE0\x00\x00\x00\x01\x00\x column=t:\x05\x00\x00, timestamp=1447900875649, value=\x0D\x91\x17s Q\x17s3!\x17sB\xD1\x17sCq\x17tD\x11\x17wD\xB1\x17y
 00\x01\x00\x00\x02\x00\x00\x02\x00\x00\x EQ\x17zX\x11\x17zj\xE1\x17z}\xB1\x17z~\xF1\x17z\x7F\x91\x17{\x801\x17~\x80\xD1\x17\x7F\x93\xA1\x17\x7F\xA6a\x17\x7F\xB
 03\x00\x00\x03\x00\x00\x04\x00\x00\x10\x 91\x17\x7F\xCC\x01\x17\x7F\xDE\xC1\x17\x7F
 00\x00\x05\x00\x00.\x00\x00\x06\x00\x00\
 x06
 \x00\xD4V7\xDC\xE0\x00\x00\x00\x01\x00\x column=t:\x05\x00\x00, timestamp=1447900875650, value=\x0D\x9B\x00\x00\x00\x00 [\x00\x00\x00\x003+\x00\x00\x00\x00E\xF
 00\x01\x00\x00\x02\x00\x00\x02\x00\x00\x B\x00\x00\x00\x00X\xBB\x00\x00\x00\x00k\x8B\x00\x00\x00\x00~[\x00\x00\x00\x00\x91\x1B\x00\x00\x00\x00\xA3\xEB\x00\x00\
 03\x00\x00\x03\x00\x00\x04\x00\x00\x13\x x00\x00\xB6\xBB\x00\x00\x00\x00\xC9\x8B\x00\x00\x00\x00\xDCK\x00\x00\x00\x00
 00\x00\x05\x00\x00/\x00\x00\x06\x00\x00\
 x06
 \x00\xD4V7\xDC\xE0\x00\x00\x00\x01\x00\x column=t:\x05\x00\x00, timestamp=1447900875649, value=\x0D\x90\x00 P\x003 \x00E\xF0\x00X\xB0\x00k\x80\x00~P\x00\x91\x1
 00\x01\x00\x00\x02\x00\x00\x02\x00\x00\x 0\x00\xA3\xE0\x00\xB6\xB0\x00\xC9\x80\x00\xDC@\x00
 03\x00\x00\x03\x00\x00\x04\x00\x00\x18\x
 00\x00\x05\x00\x00.\x00\x00\x06\x00\x00\
 x06
 \x00\xD4V7\xDC\xE0\x00\x00\x00\x01\x00\x column=t:\x05\x00\x00, timestamp=1447900875446, value=\xB6\xBB\x00\x00\x00\x00\x0D\x9B\x00\x00\x00\x00\xC9\x8B\x00\x00
 00\x01\x00\x00\x02\x00\x00\x02\x00\x00\x \x00\x00\xDCK\x00\x00\x00\x00 [\x00\x00\x00\x00E\xFB\x00\x00\x00\x003+\x00\x00\x00\x00X\xBB\x00\x00\x00\x00k\x8B\x00\x
 03\x00\x00\x03\x00\x00\x04\x00\x00\x19\x 00\x00\x00~[\x00\x00\x00\x00\x91\x1B\x00\x00\x00\x00\xA3\xEB\x00\x00\x00\x00
 00\x00\x05\x00\x00\x05\x00\x00\x06\x00\x
 00\x06
 \x00\xD4V7\xDC\xE0\x00\x00\x00\x01\x00\x column=t:\x05\x00\x00, timestamp=1447900875650, value=\x0D\x9B\x00\x00\x00\x00 [\x00\x00\x00\x003+\x00\x00\x00\x00E\xF
 00\x01\x00\x00\x02\x00\x00\x02\x00\x00\x B\x00\x00\x00\x00X\xBB\x00\x00\x00\x00k\x8B\x00\x00\x00\x00~[\x00\x00\x00\x00\x91\x1B\x00\x00\x00\x00\xA3\xEB\x00\x00\
 03\x00\x00\x03\x00\x00\x04\x00\x00\x1B\x x00\x00\xB6\xBB\x00\x00\x00\x00\xC9\x8B\x00\x00\x00\x00\xDCK\x00\x00\x00\x00
 00\x00\x05\x00\x00/\x00\x00\x06\x00\x00\
 x06
```

**Comment by juliennju** [comment]
This is bug : https://github.com/OpenTSDB/opentsdb/issues/585

**Comment by rgazaryants** [comment]
I built it from next, and made sure those changes were there..?

**Comment by manolama** [comment]
ick, ok let me double check this one post fix.

**Comment by manolama** [comment]
OK, fixed in f6c5fbbfee489c1d8f1639a64b81e2ea2028c7d5  Confirmed it myself to make sure it would work. Still need UTs around all of this functionality. Thanks!

**Comment by rgazaryants** [comment]
Yup. Looks good. Thank you!
